lib/constants/cycles.ts

```diff
@@ -1,58 +1,94 @@
+ * Year B follows year A, year C follows year B, then back again to A.
+ */
+export const SundayCycles = {
   YearA: 'YEAR_A',
   YearB: 'YEAR_B',
   YearC: 'YEAR_C',
 } as const;
 
 /**
- * Weekdays cycle that can be used as metadata for liturgical days.
+ * A dynamically generated array from {@link SundayCycles} that contains all the possible
+ * three-year Sunday cycles.
+ */
+export const SUNDAY_CYCLES = Object.values(SundayCycles);
+
+/**
+ * Represent one of the three-year Sunday cycles (available from {@link SundayCycles}).
+ */
+export type SundayCycle = typeof SUNDAY_CYCLES[number];
+
+/**
+ * A two-year cycle for the weekday mass readings (also called Cycle I and Cycle II).
+ * Odd-numbered years are the year 1 (or Cycle I); even-numbered ones are the year 2 (or Cycle II).
  */
-export const WeekdaysCycles = {
+export const WeekdayCycles = {
   Year1: 'YEAR_1',
   Year2: 'YEAR_2',
 } as const;
 
 /**
- * Psalter weeks that can be used as metadata for liturgical days.
+ * A dynamically generated array from {@link WeekdayCycles} that contains all the possible
+ * two-year weekday cycles.
  */
-export const PsalterWeeksCycles = {
-  Week1: 'WEEK_1',
-  Week2: 'WEEK_2',
-  Week3: 'WEEK_3',
-  Week4: 'WEEK_4',
-} as const;
+export const WEEKDAY_CYCLES = Object.values(WeekdayCycles);
 
 /**
- * A dynamically generated constant consisting of all the enum keys in [[LITURGICAL_DAY_CYCLES]]
+ * Represent one of the two-year weekday cycles (available from {@link WeekdayCycles}).
  */
-export const PROPER_CYCLE = Object.values(ProperCycles);
-export type ProperCycle = typeof PROPER_CYCLE[number];
+export type WeekdayCycle = typeof WEEKDAY_CYCLES[number];
 
 /**
- * A dynamically generated constant consisting of all the enum keys in [[SUNDAY_CYCLES]]
+ The four-week cycle of the psalter is coordinated with the liturgical year in such a way that
+ on the First Sunday of Advent, the First Sunday in Ordinary Time, the First Sunday of Lent,
+ and Easter Sunday the cycle is always begun again with Week 1 (others being omitted when necessary).
  */
-export const SUNDAYS_CYCLE = Object.values(SundaysCycles);
-export type SundaysCycle = typeof SUNDAYS_CYCLE[number];
+export const PsalterWeekCycles = {
+  Week1: 'WEEK_1',
+  Week2: 'WEEK_2',
+  Week3: 'WEEK_3',
+  Week4: 'WEEK_4',
+} as const;
 
 /**
- * A dynamically generated constant consisting of all the enum keys in [[WEEKDAY_CYCLES]]
+ * A dynamically generated array from {@link PsalterWeekCycles} that contains all the possible
+ * psalter week cycles.
  */
-export const WEEKDAYS_CYCLE = Object.values(WeekdaysCycles);
-export type WeekdaysCycle = typeof WEEKDAYS_CYCLE[number];
+export const PSALTER_WEEKS = Object.values(PsalterWeekCycles);
 
 /**
- * A dynamically generated constant consisting of all the enum keys in [[PSALTER_WEEKS]]
+ * Represent one of the four-week psalter cycles (available from {@link PsalterWeekCycles}).
```

Comment:
```suggestion
 * Represent one of the psalter week cycles (available from {@link PsalterWeekCycles}).
```

Reply:
Why do you suggest removing "four-week"? I think it provides a good indication that this cycle is running over four weeks.

Reply:
Based on [this comment of mine](https://github.com/romcal/romcal/pull/343#discussion_r775172676), it can be kept as is, however, I don’t think that hard-coding the number of weeks in the psalter cycle makes it better documented.

Reply:
```suggestion
 * Represent one of the four-week psalter cycle (available from {@link PsalterWeekCycles}).
```

Reply:
LGTM.